### ISSUE TITLE
loss calculation

### ISSUE THREAD

**Issue by TaeheeJeong** [kentaroy47/frcnn-from-scratch-with-keras#63]
instead of using custom loss function, 
can you write a code with keras loss function?

**Comment by kentaroy47** [comment]
https://github.com/kentaroy47/frcnn-from-scratch-with-keras/blob/master/keras_frcnn/losses.py

I understand that the custom loss function is hard to decode.
but not sure if you can write binding box regression loss with the keras loss.
